wgengine/userspace.go

```diff
@@ -1487,8 +1485,7 @@ func (ls fwdDNSLinkSelector) PickLink(ip netip.Addr) (linkName string) {
 }
 
 var (
-	metricMagicDNSPacketIn = clientmetric.NewCounter("magicdns_packet_in") // for 100.100.100.100
```

Comment:
_Should_ we be incrementing this someplace?

Reply:
We have this whole pile of dns-related metrics: https://github.com/tailscale/tailscale/blob/cbeb8e7422fd61413cbab419e4453477b1bb7a38/net/dns/resolver/tsdns.go#L1285
We count queries, which seems more practical than individual packets.
I think that's good enough?